moveit_planners/pilz_industrial_motion_planner/test/unit_tests/src/unittest_trajectory_functions.cpp

```diff
@@ -167,6 +196,43 @@ bool TrajectoryFunctionsTestBase::tfNear(const Eigen::Isometry3d& pose1, const E
   return true;
 }
 
+bool TrajectoryFunctionsTestBase::jointsNear(const std::vector<double>& joints1, const std::vector<double>& joints2,
+                                             double epsilon)
+{
+  if (joints1.size() != joints2.size())
+  {
+    return false;
+  }
+  for (std::size_t i = 0; i < joints1.size(); i++)
```

Comment:
Tend to use `++i` instead of `i++` in loops

Reply:
Done :+1: